e2e/cypress/support/api/role.js

```diff
@@ -166,6 +164,19 @@ export const defaultRolesPermissions = {
         'sysconsole_write_experimental_bleve',
         'purge_bleve_indexes',
         'post_bleve_indexes',
+        'sysconsole_read_compliance_data_retention',
+        'sysconsole_read_compliance_compliance_export',
+        'sysconsole_read_compliance_compliance_monitoring',
+        'sysconsole_read_compliance_custom_terms_of_service',
+        'sysconsole_write_compliance_data_retention',
+        'sysconsole_write_compliance_compliance_export',
+        'sysconsole_write_compliance_compliance_monitoring',
+        'sysconsole_write_compliance_custom_terms_of_service',
+        'create_data_retention_job',
+        'read_data_retention_job',
+        'create_compliance_export_job',
+        'read_compliance_export_job',
+        'read_audits',
```

Comment:
Note to self: Review and verify these after Server PR has been fixed to pass build.